model_compression_toolkit/core/common/quantization/quantization_params_generation/error_functions.py

```diff
@@ -143,6 +144,36 @@ def _kl_error_function(x: np.ndarray,
                                range_max=range_max)
 
 
+def _kl_error_function_wrapper(x: np.ndarray,
+                               range_min: np.ndarray,
+                               range_max: np.ndarray,
+                               n_bins: int = 2048,
+                               n_bits: int = 8) -> np.ndarray:
+    """
+    Compute the error function between a tensor to its quantized version per channel.
```

Comment:
Computes the error function between a tensor and its quantized version for each channel.
    The error is based on the KL-divergence between the distributions.
    The function uses a specified number of bins to compute the histogram of the float tensor.
    It requires the threshold and number of bits used for quantization to determine the histogram's boundaries and the number of quantized bins.

Reply:
I didn't change the descriptions in this PR, but I can change them.